### PR TITLE
[src] Add a global namespace using for a couple of namespaces.

### PR DESCRIPTION
**Pull request by rolfbjarne**
Also remove all the 'using System.Runtime.Versioning;' statements everywhere.

### CODE REVIEW

**Comment by rolfbjarne** [comment]
@chamons this might make #14287 somewhat smaller, it goes in first (or I can wait until you've merged #14287).

**Review by chamons — APPROVED** [review]

**Comment by chamons** [comment]
If you get yours green before I finish fixing the last test and writing one last one (very likely) go ahead. It should be a trivial resolve.

**Review by dalexsoto — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

**Comment by rolfbjarne** [comment]
Test failure is unrelated (https://github.com/xamarin/maccore/issues/1083).

**Pull request merged**
